ext/legacycustomsearches/CRM/Contact/Form/Search/Custom/FullText.php

```diff
@@ -372,16 +372,15 @@ public function summary() {
     }
     $dao = CRM_Core_DAO::executeQuery($sql);
 
-    $activityTypes = CRM_Core_PseudoConstant::activityType(TRUE, TRUE);
     $roleIds = CRM_Event_PseudoConstant::participantRole();
     while ($dao->fetch()) {
       $row = [];
       foreach ($this->_tableFields as $name => $dontCare) {
-        if ($name != 'activity_type_id') {
+        if ($name !== 'activity_type_id') {
           $row[$name] = $dao->$name;
         }
         else {
-          $row['activity_type'] = $activityTypes[$dao->$name] ?? NULL;
+          $row['activity_type'] = CRM_Core_PseudoConstant::getLabel('CRM_Activity_BAO_Activity', 'activity_type_id', $dao->$name);
```

Comment:
This isn't the same. See https://github.com/civicrm/civicrm-core/pull/23983#discussion_r917453363

Reply:
@demeritcowboy this seems qualitively different to the original though - I think we expect custom field options to be disabled & enabled & existing rows to be fine with that - but do we really think it's 'OK' to disable an activity type when activities of that type exist and then display them (without the type) - I think implicit in activity type being a required field is that the type should not be disable-able if in use

Reply:
I would agree with you, but 50% of people will disagree with it.

Reply:
Specifically on activity_type?

Reply:
I'll add a note to https://lab.civicrm.org/dev/core/-/issues/19.

Reply:
OK - I'll close this & let it resolve itself on gitlab before putting more on github